app/Actions/WordPress/AuthorizeEquipmentAction.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Actions\WordPress;
+
+use App\External\WooCommerce\Api\WooCommerceApi;
+use Spatie\QueueableAction\QueueableAction;
+use Illuminate\Support\Facades\Log;
+
+class AuthorizeEquipmentAction
```

Comment:
I don't add "Action" at the end of my actions. They're in the action namespace and there's not much gained by calling it an `Action`. Anyone who uses it will essentially already know. Even in use, it doesn't add much readability. I would have called this one `AuthorizeEquipmentUse` personally, because it makes it easier to distinguish we're authorizing the use of the equipment vs authorizing a piece of equipment to be used somehow.

```php
app(AuthorizeEquipmentAction::class)->onQueue()->execute($actorId, $person->id, $equipment->user_plan_id);
```
vs
```php
app(AuthorizeEquipmentUse::class)->onQueue()->execute($actorId, $person->id, $equipment->user_plan_id);
```

Reply:
> I don't add "Action"...

I don't see why not. We have `SlackEventController` (it's a `Controller`), `AppServiceProvider` (It's a `Provider`), `MembershipOptionsModal` (It's a `Modal`), and the list goes on... 

IMO it helps orient readers to what part of the app they're in and reinforces their understanding of the architecture (eg: what happens in Controllers vs Actions) that might otherwise be lost to new users.

Given that this project is open source and member driven, I think we should be writing for newcomers (to the app, but also to PHP/Laravel) rather than to experienced users of the framework.

That said, I'm not going to die on this hill. If you want me to change it, I'll change it.

Reply:
In all of the other examples those are what things "are" vs an action which is closer to a verb. ie I consider it closer to a method name. That's the only reason I consider actions differently. Almost everywhere else, I tend to agree with you.